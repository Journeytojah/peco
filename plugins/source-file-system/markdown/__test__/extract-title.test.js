const markdown = require('markdown-it')
const plugin = require('../extract-title')

test('main', () => {
  const md = markdown()
  const env = {}
  md.use(plugin)
  const html = md.render(
    `
# hello __world__

# hoo

wow
  `,
    env
  )

  expect(env).toMatchSnapshot('env')
  expect(html).toMatchSnapshot('html')
})
