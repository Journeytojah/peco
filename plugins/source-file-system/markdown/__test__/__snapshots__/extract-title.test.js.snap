// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main: env 1`] = `
Object {
  "title": "hello __world__",
}
`;

exports[`main: html 1`] = `
"
<h1>hoo</h1>
<p>wow</p>
"
`;
